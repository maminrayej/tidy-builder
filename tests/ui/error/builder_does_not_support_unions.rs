#[derive(tidy_builder::Builder)]
union MyUnion {
    f1: u32,
    f2: f32,
}

fn main() {}