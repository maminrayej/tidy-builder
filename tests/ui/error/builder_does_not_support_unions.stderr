error: Builder does not support unions
 --> tests/ui/error/builder_does_not_support_unions.rs:2:1
  |
2 | union MyUnion {
  | ^^^^^
